etc/template/Occlum.json

```diff
@@ -61,14 +61,6 @@
             "target": "/host",
```

Comment:
Suggest removing the /host from the template too. The host FS could be an option for the user. The user needs to add it by himself. The reason is that: The hacker may put some script in the host FS and execute it somehow. Or the hacker may place a soft link in the host FS, which may lead to unexpected behavior.

Reply:
@guzongmin Even if removing the /host from the tempate, the building generated internal .Occlum_sys.json still has the default hostfs mount information in current solution. Are you suggesting that unless user explictely  setting the /host mount, no default hostfs mount should be enabled in building process?

Reply:
@guzongmin I fully agree that If "Unless user explicitly set the /host mount, no default hostfs mounted" from security pespective. But perhaps some previous projects may meet problems because they thought that there was a default hostfs mounted. I will discuss with the team.